addons/better-featured-project/addon.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Profile page banner",
+  "description": "Makes your featured project into a nice-looking banner for your profile.",
```

Comment:
The description appears to imply the addon changes what other users see, instead of what the extension user sees.

Reply:
> Makes the featured project on any user's profile page into a nice-looking banner.
Hopefully this is better?